### ISSUE TITLE
Diffrent styling for the slider module

### ISSUE THREAD

**Issue by CmdrMahesh** [fomantic/Fomantic-UI#1625]
# Feature Request
I am using the slider module and it works perfectly and I wonder if there is any way to change the position of the labels being displayed on the slider.

For example, I want something like this:
![request](https://user-images.githubusercontent.com/46396430/89045797-c0545300-d369-11ea-8e0c-856662182f79.png)

And we have this:
![given](https://user-images.githubusercontent.com/46396430/89045832-d104c900-d369-11ea-8b34-54b7c996dd6e.png)

I have tried some modifications at one of my demo servers and here is the [link](https://demo.woochoiceplugin.com/hify-store/product-category/eo_diamond_shape_cat/?EO_WBC=1&BEGIN=eo_diamond_shape_cat&STEP=1&FIRST=&SECOND=) to the page.

![having](https://user-images.githubusercontent.com/46396430/89046646-e0384680-d36a-11ea-9c68-941a541c64d0.png)

On the page, I have a filter slider named `SYMM.` which has a `Very Good` label, and unfortunately, my CSS skill is not so good that's why it been overlapped by the slider nodes.

So to avoid my confusing CSS codes I like to ask if there is any way to have the slider as I want?

**Comment by ko2in** [comment]
You can override as following in `slider.overrides`:
```
.ui.labeled.slider > .labels .label {
    display: inline-flex;
    position: absolute;
    transform: translate(-50%, -100%);
    white-space: nowrap;
}
```
Here's the overrides for slider in default theme (https://github.com/fomantic/Fomantic-UI/blob/develop/src/themes/default/modules/slider.overrides).

Here's the result with `white-space: nowrap`.
![FUI](https://user-images.githubusercontent.com/930315/89049321-17acf000-d377-11ea-9485-d6dea7243dce.png)

Of course, long label is not good to use when you've too many options with narrow spaces between them.

**Comment by CmdrMahesh** [comment]
@ko2in , Thank you for the fast reply.

I will try this one and update you by tomorrow morning.

Once again, thanks.

**Comment by exoego** [comment]
I think this adding this variation is valuable, since long text label like this poll seems very common.

**Comment by ko2in** [comment]
Yes. It sounds reasonable. Will you prepare PR? Or should I submit this?

**Comment by jamessampford** [comment]
Couldn't we add a class to the labels to allow them to appear above or below the slider, rather than using override?

Something like `.labels.bottom.aligned`? Could we also potentially apply to a label itself, so the labels could potentially rotate (if wanted) to be above or below, eg `.labels .label.bottom.aligned` (top/above alignment could be default for comparability)

**Comment by CmdrMahesh** [comment]
@jamessampford The class for aligning the label at either top or the bottom sounds an extremely good idea and that will be much easy for backend devs like me.

**Comment by CmdrMahesh** [comment]
Thank you guys, that one fixed my issue and I wonder if there is some way to avoid CSS of other libraries being effecting the CSS of the FU/SU.

Anyway, you guys were helpful, and adding PR will surely helpful to others in the future.

**Comment by lubber-de** [comment]
Implemented by #1631
